packages/svelte/src/lib/components/Form/Radio/Radio.svelte

```diff
@@ -85,22 +65,32 @@
   let labelClasses = `label ${readOnly ? 'readonly' : ''} 
                             ${disabled ? 'disabled' : ''}`;
   let descriptionClasses = `description ${fontSizeClass}`;
+
+  let inputClasses = `input ${readOnly ? 'readonly' : ''} 
+                            ${disabled ? 'disabled' : ''}`;
+
+  const slugify = (str = '') =>
+    str.toLowerCase().replace(/ /g, '-').replace(/\./g, '');
 </script>
 
 <div
   class={`${containerClasses} ${fontSizeClass}`}
-  on:click={toggle}
-  on:keydown={handleKeydown}
   tabindex="-1"
   role="radio"
   aria-checked={checked}
-  aria-label="Select an option"
+  aria-label={label}
   aria-labelledby={description}
+  id={`radio-${radioGroup.uniqueId}`}
 >
   <span class={`control radio`}>
     <input
-      class="input"
+      class={inputClasses}
       type="radio"
+      id={slugify(label)}
```

Comment:
Hvorfor er det nødvendig å bruke `slugify` her? Kan den ikke bare bruke en uuid heller? 
Samme gjelder for `label` sin `for` attribute (Men da samme ID som input, så klart)

Det kan potensielt bli problematisk om to elementer har samme label og da får samme ID, selv om det er usannsynlig

Reply:
Enig, var litt sen kveldskode der ser jeg... Jobber med en bedre løsning nå (uten slugify).